crates/oci/src/loader.rs

```diff
@@ -85,11 +82,24 @@ impl OciLoader {
                         format!("failed to create temporary mount path {mount_path:?}")
                     })?;
 
-                // Copy content
-                // TODO: parallelize
-                tokio::fs::copy(&content_path, &mount_path)
-                    .await
-                    .with_context(|| format!("failed to copy {content_path:?}->{mount_path:?}"))?;
+                if let Some(content_bytes) = file.content.inline.as_deref() {
+                    // Write inline content to disk
```

Comment:
Is here where we'd want to check that `file.content.inline` matches `file.content.digest`, if the latter is set?